src/utils/utility.ts

```diff
@@ -104,3 +104,11 @@ export function camelCasePropertyNames(obj) {
 export function toISOFormat(datetime) {
   return moment(datetime).format("YYYY-MM-DDTHH:mm:ss.SSSSSZ");
 }
+
+export function tryCatchCall(fn, thisArg, ...args) {
+  try {
+    return fn.call(thisArg, ...args);
+  } catch (e) {
+    console.warn(e);
+  }
+}
```

Comment:
This is an odd function, I recommend silencing explicitly where needed.

Reply:
I removed it. Originally, it was meant to silence GA analytics plugin error on invalid trackingID, but it turned out that the plugins are initialized asynchronously, so no need for the try/catching.